Sources/libhostmgr/libhostmgr.swift

```diff
@@ -317,3 +320,17 @@ func applyVMSettings(_ settings: [StoppedVM.VMOption], to parallelsVM: StoppedVM
         Console.warn("Unable to remove device: \(error.localizedDescription)")
     }
 }
+
+/// Force kill all running VM processes
+///
+/// We have a relatively frequent error in CI jobs:
+/// "Failed to unregister the VM: Unable to perform the action because the
+/// virtual machine is busy. The virtual machine is currently running.
+/// Please try again later."
+///
+/// This functions kills all the running VM processes so that we can unregister it later.
+func killVMProcesses() {
+    Console.info("Killing running virtual machines")
+    // The command failure is ignored because there may not be any running VM.
+    _ = try? shellOut(to: "pkill", arguments: ["-9", "-f", #"Parallels VM\.app/Contents/MacOS/prl_vm_app"#])
```

Comment:
idk how `shellOut` works under the hood but do you need to actually escape the `\.` in there, doesn't it do it for you (especially when you already pass it the commands as individual arguments and not a single big string)?

i.e. wouldn't this be enough?
```suggestion
    _ = try? shellOut(to: "pkill", arguments: ["-9", "-f", "Parallels VM.app/Contents/MacOS/prl_vm_app"])
```

Reply:
The string (`pkill` argument) is a regex. The `\` turn `.` which matches any character into a literal `.`. That's why I used the `#"..."` string literal, otherwise there needs to be two `\` in the string which looks less nice...

Reply:
Oh got it 👍